components/datetime/src/metazone.rs

```diff
@@ -28,22 +27,19 @@ impl MetaZoneCalculator {
     /// use icu::datetime::metazone::MetaZoneCalculator;
     ///
     /// let provider = icu_testdata::get_provider();
-    /// let mzc = MetaZoneCalculator::new(
-    ///     locale!("en"),
-    ///     &provider,
-    /// );
+    /// let mzc = MetaZoneCalculator::try_new(&provider);
     ///
     /// assert!(mzc.is_ok());
     /// ```
-    pub fn new<L, ZP>(locale: L, zone_provider: &ZP) -> Result<Self, DateTimeFormatterError>
+    pub fn try_new<P>(
```

Comment:
You might forget to update the doc on line 14.

Reply:
Yep; fixed in https://github.com/unicode-org/icu4x/pull/2274/commits/8d2ea5e9ccc1936846b9797d32bcdb88c0a01df5